# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../gle/no_debugout.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./src/port.h.orig	Mon Jul 29 12:03:30 2002
+++ ./src/port.h	Mon Jul 29 12:21:06 2002
@@ -84,7 +84,7 @@
 
 /* ====================================================== */
 
-#if DEBUG_OUTPUT
+#if 0 /* DEBUG_OUTPUT */
 #undef GL_32
 #undef OPENGL_10
 
